math/gts/Portfile

```diff
@@ -4,8 +4,7 @@ PortSystem          1.0
 PortGroup           muniversal 1.0
 
 name                gts
-version             0.7.6
-revision            3
+version             121130
```

Comment:
We usually use an 8-digit `YYYYMMDD` year when making up a version number, and `gts-config --version` still identifies this as `0.7.6`, so I would like to see this `version` field changed to `0.7.6-20121130`, and similar changes for the other ports' versions. This will make it easier to go back to the real version numbering scheme once they release the next stable version, without having to add an `epoch`.

The libgts major, current and compatibility versions have not changed, but since this is a snapshot, I don't know whether the library has actually changed and they just haven't bumped the versions yet... If the library has changed, we'd need to revbump anything using it.

Reply:
This should now be fixed.
I do not think revbumps are necessary.
Both versions of GTS install `libgts-0.7.5.dylib`, and nothing seems to depend on Gerris.